cadquery/occ_impl/shapes.py

```diff
@@ -2558,6 +2645,7 @@ def fillet(
         """
         Apply 2D or 3D fillet to a wire
 
+        :param wire: The input wire to fillet. Currently only open wires are supported
```

Comment:
This docstring line was previously removed in #1630.  It looks like it was inadvertently added back.

Reply:
I merged master, let's see if all is fine.